agentforce/1.0/modules/ROOT/pages/agentforce-connector-studio.adoc

```diff
@@ -0,0 +1,148 @@
+= Using Anypoint Studio to Configure MuleSoft Chain AI Connector 1.0 - Mule 4
+:exchange-search-name: mulesoft ai chain
+:connector-name: Agentforce
+:lc-connector-name: agentforce
+:studio-connector-name: Agentforce Connector
+
+include::connectors::partial$t-studio-partial.adoc[tags=topic-intro]
+
+
+[[create-mule-project]]
+== Create a Mule Project
+
+include::connectors::partial$t-studio-partial.adoc[tags=create-project-intro]
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Agentforce Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and add the required dependencies to the project's `pom.xml` file:
+
+. In *Mule Palette*, click *(X) Search in Exchange*.
```

Comment:
Note: This experience will change once the Agentforce Connector is bundled with ACB